src/navigation/useDeepLinking.js

```diff
@@ -36,7 +36,8 @@ function getSettingsScreen(pathname) {
 
 export function useDeepLinkHandler() {
   const { navigate } = useNavigation();
-  const { manifests } = usePlatformApp();
+  const { state } = useRemoteLiveAppContext();
+  const manifests = state.value.liveAppByIndex;
```

Comment:
If the provided it not loaded at this time, value will be undefined. Make sure the data is loaded by checking value is defined

Reply:
fix uploaded